### DISCUSSION TITLE
Premiere Pro XML Customisation

### DISCUSSION THREAD

**Discussion by AidenSoave** [WyattBlue/auto-editor#466] [1. Help]
Any way for us to be able to have silent portions customised in some way. for example
silent portions on track 2.  whereas track 1 has the regular portions.
or if you can have silent portions be labeled a different colour. than regular portions.
or even have silent parts still open but as empty spaces not as clips

**Comment by dr04e606** [comment]
It would be great if Auto-Editor had an option to create an XML export where silence is not removed, but instead placed on separate tracks. This would make it easier to see exactly what is identified as "silence".

**Reply by WyattBlue** [reply]
You can do `--video-speed 1 --silent-speed 1` and that will split the clips without cutting.

**Reply by dr04e606** [reply]
Thanks! That works. 

However, I would prefer it if the silent parts were on a separate track with a different label color. 
![image](https://github.com/user-attachments/assets/36f88049-ac32-41c8-afcb-d2ce52088aa2)
